src/main/java/uk/gov/hmcts/darts/cases/mapper/HearingEntityToCaseHearing.java

```diff
@@ -38,7 +39,11 @@ private Hearing mapToHearing(HearingEntity entity) {
         var transcripts = entity.getTranscriptions()
             .stream()
             .filter(transcriptionEntity -> BooleanUtils.isTrue(transcriptionEntity.getIsManualTranscription())
-                || StringUtils.isNotBlank(transcriptionEntity.getLegacyObjectId()))
+                || StringUtils.isNotBlank(transcriptionEntity.getLegacyObjectId())
+            )
+            .filter(transcriptionEntity -> transcriptionEntity.getTranscriptionDocumentEntities().isEmpty()
+                || transcriptionEntity.getTranscriptionDocumentEntities().stream().noneMatch(TranscriptionDocumentEntity::isHidden)
```

Comment:
Should super users still be able to view these?

Reply:
No, super users aren't able to see manual deletion related things.